oap-server/server-starter/src/main/resources/oal/core.oal

```diff
@@ -19,15 +19,15 @@
 // Service scope metrics
 service_resp_time = from(Service.latency).longAvg();
 service_sla = from(Service.*).percent(status == true);
-service_cpm = from(Service.*).cpm();
+service_cpm = from(Service.*).cpm5d();
```

Comment:
```suggestion
service_cpm5d = from(Service.*).cpm5d();
```

Let's follow this to update settings. @wankai123 Once this is merged, Grafana setup should be updated according on showcase.